cms-storyblok/src/render-blocks.js

```diff
@@ -76,14 +76,14 @@ const components = {
     </script>`;
   },
   container(blok) {
-    let classes = "";
+    let attributes = 'is-staggered="true"';
 
     if (blok.isDarkTheme) {
-      classes += 'is-dark="true"';
+      attributes += ' is-dark="true"';
```

Comment:
@Paul-Hebert If you find yourself forgetting to add the extra space prefix each time a new attribute is added, an alternative way to handle this might be something like:

```js
const attributes = ['is-staggered="true"'];

if (blok.isDarkTheme) {
  attributes.push('is-dark="true"'); // notice I don't need to add the space prefix
}

// Join with a space here instead
<c4-container ${attributes.join(' ')}>
```

It's up to you whether or not this change makes it less prone to error or not. 🙂

Reply:
That's a good idea :+1: